src/DataService/DataService.php

```diff
@@ -966,7 +966,7 @@ public function SendEmail($entity, $email = null)
      * @param int $maxResults Page size
      * @return array Returns an array of entities fulfilling the query. If the response is Empty, it will return NULL
```

Comment:
Could you add a line describing `$includes` in the docblock?

Reply:
done!